### PR TITLE
CNV-12268: Adding new storage feature to RN

### PR DESCRIPTION
**Pull request by bgaydosrh**
for 4.8 only.

Jira: https://issues.redhat.com/browse/CNV-12268

Tagging @aglitke for code review.
Tagging @duyanyan for QE review.

Direct doc preview link: https://deploy-preview-34234--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#virt-4-8-storage-new

### CODE REVIEW

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -63,12 +63,12 @@ The SVVP Certification applies to:
...
 
 //CNV-11390 OpenShift Virtualization now supports live-migration of VM connected to an SR-IOV network.
-* {VirtProductName} now supports xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[live migration of virtual machines] that are attached to an SR-IOV network interface if the `sriovLiveMigration` feature gate is enabled in the `HyperConverged` custom resource (CR). 
+* {VirtProductName} now supports xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[live migration of virtual machines] that are attached to an SR-IOV network interface if the `sriovLiveMigration` feature gate is enabled in the `HyperConverged` custom resource (CR).
 
 [id="virt-4-8-storage-new"]
 === Storage
 
-//CNV-12268 Cloning a DataVolume into a different namespace (ie. when creating a VM from a template) is now faster and more efficient when using storage that supports CSI Snapshots.
+* Cloning a data volume into a different namespace is now faster and more efficient when using storage that supports the Containerized Snapshot Interface (CSI). Using CSI snapshots improves performance when creating a virtual machine from a template.
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc]: Just a  tweak to: * Cloning a data volume into a different namespace is now faster and more efficient when using storage that supports Container Storage Interface (CSI) Snapshots. CDI automatically uses CSI Snapshots when they are available to improve performance when you create a virtual machine from a template.

If this is OK, I'll send it on to Peer Review.

**Comment by duyanyan** [comment]
LGTM

**Review by aglitke — APPROVED** [review]

**Review by kgoldbla — COMMENTED** [review]
That looks fine

File: virt/virt-4-8-release-notes.adoc [hunk 2]
```diff
@@ -63,12 +63,12 @@ The SVVP Certification applies to:
...
-* {VirtProductName} now supports xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[live migration of virtual machines] that are attached to an SR-IOV network interface if the `sriovLiveMigration` feature gate is enabled in the `HyperConverged` custom resource (CR). 
+* {VirtProductName} now supports xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[live migration of virtual machines] that are attached to an SR-IOV network interface if the `sriovLiveMigration` feature gate is enabled in the `HyperConverged` custom resource (CR).
 
 [id="virt-4-8-storage-new"]
 === Storage
 
-//CNV-12268 Cloning a DataVolume into a different namespace (ie. when creating a VM from a template) is now faster and more efficient when using storage that supports CSI Snapshots.
+* Cloning a data volume into a different namespace is now faster and more efficient when using storage that supports Container Storage Interface (CSI) Snapshots. CDI automatically uses CSI Snapshots when they are available to improve performance when you create a virtual machine from a template.
```
Comment (sjhala-ccs) [inline virt/virt-4-8-release-notes.adoc:71]: s/Snapshots/snapshots

Also, you might want to spell out CDI and add commas - "The Containerized Data Importer uses CSI snapshots, when they are available, to improve performance when you create a virtual machine from a template."

**Review by sjhala-ccs — COMMENTED** [review]
Just a couple of suggestions, otherwise looks good!

File: virt/virt-4-8-release-notes.adoc (hunk 2 above)
Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc:77]: Done.

**Review by ousleyp — APPROVED** [review]
lgtm

**Pull request merged**